docs/readthedocs/source/doc/Nano/Overview/cuda_patch.md

```diff
@@ -0,0 +1,56 @@
+# CUDA Patch
+
+BigDL-Nano also provides CUDA patch to help you run CUDA code without GPU. This patch will replace CUDA operations with equivalent CPU operations, so after applying it, you can run CUDA code on your CPU without changing any code.
+
+This patch is included in `bigdl.nano.pytorch.patch_torch`, you can call `patch_torch(cuda_to_cpu=True)` to enable it (the `cuda_to_cpu` parameter defaults to `True`). You can also use it alone by `bigdl.nano.pytorch.patching.gpu_cpu.patch_cuda`.
```

Comment:
Should we mention a little bit here or somewhere what else is included in `patch_torch`? The first time I read the paragraph here, it is not so clear for me what is the difference between `patch_torch` and `patch_cude`.

Reply:
junwei will add key feature of `patch_torch` in the future, we can simply add a link to it after he added